### DISCUSSION TITLE
You Dont have Write Access to this Github Repository , While Trying to Push my code from UiPath

### DISCUSSION THREAD

**Discussion by Apollo27296** [community/community#22406] [New to GitHub]
<p>“You Dont have Write Access to this Github Repository , While Trying to Push my code from UiPath”. I’m getting this message while sigining in with my github credentials , but if i use Personal Access Token. I dont face any issues while Push my Code. I want to use My username &amp; password instead</p>
<p>GitHub: LibGit2Sharp.LibGit2SharpException: request failed with status code: 403<br>
at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)<br>
at LibGit2Sharp.Core.Proxy.git_remote_push(RemoteHandle remote, IEnumerable<code>1 refSpecs, GitPushOptions opts)    at LibGit2Sharp.Network.Push(Remote remote, IEnumerable</code>1 pushRefSpecs, PushOptions pushOptions)<br>
at LibGit2Sharp.Network.Push(Remote remote, String pushRefSpec, PushOptions pushOptions)<br>
at UiPath.Studio.Plugin.Git.Services.PushService.&lt;&gt;c__DisplayClass2_0.b__0()<br>
at UiPath.Studio.Plugin.Git.Desktop.Services.UiAuthenticationSession.Execute[T](ExecuteArgs`1 args), HResult -2146233088</p>
<p>This is the log file generated in UiPath while pushing the code</p>

**Comment by jsoref** [comment, marked as answer]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">

      <a href="https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/" target="_blank" rel="noopener nofollow ugc" title="08:28PM - 15 December 2020">The GitHub Blog – 15 Dec 20</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/2119212/181070783-ff40eab6-053c-4c2b-aee6-257b4ec43f64.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11f8abd6e6e16e4af4babafe405da97b4856127a_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11f8abd6e6e16e4af4babafe405da97b4856127a_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/1/11f8abd6e6e16e4af4babafe405da97b4856127a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11f8abd6e6e16e4af4babafe405da97b4856127a_2_10x10.png"></div>

<h3><a href="https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/" target="_blank" rel="noopener nofollow ugc">Token authentication requirements for Git operations | The GitHub Blog</a></h3>

  <p>Beginning August 13th, 2021, we will no longer accept account passwords when authenticating Git operations on GitHub.com.</p>

  <p>
    <span class="label1">Est. reading time: 3 minutes</span>
  </p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
